Ficha9/grafo_Ndirigidos.c

```diff
@@ -0,0 +1,262 @@
+/*****************************************************************/
+/*   Grafo c/ matriz de adjacencias | PROG2 | MIEEC | 2018/19    */      
+/*****************************************************************/
+
+#include <stdio.h>
```

Comment:
It seems the improvements done on the other implementation have not been done here.

Being a non directed graph will allow some simplications in some of the functions.

Reply:
Ok, I will try to improve that.